core/services/offchainreporting2/validate_test.go

```diff
@@ -94,69 +92,17 @@ chainID = 1337
 			assertion: func(t *testing.T, os job.Job, err error) {
 				require.NoError(t, err)
 				assert.Equal(t, 1, int(os.SchemaVersion))
-				assert.False(t, os.Offchainreporting2OracleSpec.IsBootstrapPeer)
 			},
 		},
 		{
-			name: "decodes bootstrap toml",
+			name: "empty pipeline string",
 			toml: `
 type               = "offchainreporting2"
 schemaVersion      = 1
 relay              = "evm"
 contractID    = "0x613a38AC1659769640aaE063C651F48E0250454C"
 p2pPeerID          = "12D3KooWHfYFQ8hGttAYbMCevQVESEQhzJAqFZokMVtom8bNxwGq"
 p2pBootstrapPeers  = []
-isBootstrapPeer    = true
-[relayConfig]
-chainID = 1337
-`,
-			assertion: func(t *testing.T, os job.Job, err error) {
-				require.NoError(t, err)
-				assert.Equal(t, 1, int(os.SchemaVersion))
-				assert.True(t, os.Offchainreporting2OracleSpec.IsBootstrapPeer)
-			},
-		},
-		{
-			name: "raises error on extra keys",
-			toml: `
-type               = "offchainreporting2"
-schemaVersion      = 1
-relay              = "evm"
-contractID    = "0x613a38AC1659769640aaE063C651F48E0250454C"
-p2pPeerID          = "12D3KooWHfYFQ8hGttAYbMCevQVESEQhzJAqFZokMVtom8bNxwGq"
-p2pBootstrapPeers  = [
-"12D3KooWHfYFQ8hGttAYbMCevQVESEQhzJAqFZokMVtom8bNxwGq@127.0.0.1:5001",
-]
-isBootstrapPeer    = true
-ocrKeyBundleID        = "73e8966a78ca09bb912e9565cfb79fbe8a6048fab1f0cf49b18047c3895e0447"
-monitoringEndpoint = "chain.link:4321"
-transmitterID = "0xF67D0290337bca0847005C7ffD1BC75BA9AAE6e4"
-observationTimeout = "10s"
-observationSource = """
-ds1          [type=bridge name=voter_turnout];
-ds1_parse    [type=jsonparse path="one,two"];
-ds1_multiply [type=multiply times=1.23];
-ds1 -> ds1_parse -> ds1_multiply -> answer1;
-answer1      [type=median index=0];
-"""
-[relayConfig]
-chainID = 1337
-`,
-			assertion: func(t *testing.T, os job.Job, err error) {
-				require.Error(t, err)
-				assert.Contains(t, err.Error(), "unrecognised key for bootstrap peer: observationSource")
-			},
-		},
-		{
-			name: "empty pipeline string non-bootstrap node",
-			toml: `
-type               = "offchainreporting2"
-schemaVersion      = 1
-relay              = "evm"
-contractID    = "0x613a38AC1659769640aaE063C651F48E0250454C"
-p2pPeerID          = "12D3KooWHfYFQ8hGttAYbMCevQVESEQhzJAqFZokMVtom8bNxwGq"
-p2pBootstrapPeers  = []
-isBootstrapPeer    = false
```

Comment:
Feels like we lost a test case here?

Reply:
The test was for invalid keys for bootstrap jobs. As OCR2 doesn't have bootstrap jobs anymore the test is unneeded.